lib/main_micro_alp.pm

```diff
@@ -14,7 +14,7 @@ use Exporter;
 use main_common;
 use main_containers qw(load_container_tests is_container_test);
 use testapi qw(check_var get_required_var get_var set_var);
-use version_utils qw(is_microos is_sle_micro is_leap_micro is_alp is_staging is_released is_transactional is_rt);
+use version_utils qw(is_microos is_sle_micro is_leap_micro is_alp is_staging is_released is_transactional is_rt is_public_cloud);
```

Comment:
just my IMHO not necessary to follow - when list getting so big it is better just to do 
```suggestion
use version_utils;
```